addons/attributes/initKeybinds.sqf

```diff
@@ -0,0 +1,34 @@
+[ELSTRING(main,DisplayName), QGVAR(open), [LSTRING(OpenAttributesDisplay), LSTRING(OpenAttributesDisplay_Description)], {
+    if (!isNull curatorCamera && {!dialog && {!GETMVAR(RscDisplayCurator_search,false)}}) then {
+        curatorSelected params ["_objects", "_groups", "_waypoints", "_markers"];
+
+        private _mouseOver = if (count curatorMouseOver > 1) then {
+            curatorMouseOver params ["_type", "_entity", ["_waypoint", -1]];
+            switch (true) do {
+                case ([_entity, _waypoint] in _waypoints): {
+                    [_entity, _waypoint]
+                };
+                case (_entity in _objects);
+                case (_entity in _groups);
+                case (_entity in _markers): {_entity};
+                default {nil};
+            };
+        };
+        if (!isNil "_mouseOver") exitWith {
+            _mouseOver call BIS_fnc_showCuratorAttributes;
+        };
```

Comment:
I think this is better written as:

```sqf
scopeName "Main";

curatorMouseOver params ["_type", "_entity", "_index"];

if (_type != "") then {
    if (_type == "ARRAY") then {
        _entity = [_entity, _index];
    };

    if (curatorSelected findIf {_entity in _x} != -1) then {
        _entity call BIS_fnc_showCuratorAttributes;
        breakOut "Main";
    };
};
```

Reply:
https://github.com/zen-mod/ZEN/pull/437/commits/e37d2ec65ca358f42f5f0c61e689bb5b9a66152b